flexmeasures/data/models/forecasting/model_spec_factory.py

```diff
@@ -27,6 +41,81 @@
 """
 
 
+logger = logging.getLogger(__name__)
+
+
+class TBSeriesSpecs(SeriesSpecs):
+
+    """Define how to collect timely beliefs.
+    This sets up a class which to call `collect` on, together with parameters to pass.
```

Comment:
I'd prefer such a docstring:

"Compatibility for using timetomodel.SeriesSpecs or timely_beliefs.BeliefsDataFrames.

This implements `_load_series` such that `TimedValue.collect` is called on the generic asset class, with the parameters in `collect_params`. Obviously,  that `collect` function should return a BeliefsDataFrame.
"